usr/share/rear/wrapup/default/990_copy_logfile.sh

```diff
@@ -29,17 +29,24 @@ recovery_system_roots_home_dir=$TARGET_FS_ROOT/root
 test -d $recovery_system_roots_home_dir || mkdir $verbose -m 0700 $recovery_system_roots_home_dir >&2
 ln -s $recover_log_dir/$final_logfile_name $recovery_system_roots_home_dir/rear-$( date -Iseconds ).log || true
 
+# Copy backup restore related files (in particular the backup restore log file) if exists.
+# This will be done as the last one of the exit tasks of this script because
+# the exit tasks are executed in reverse ordering of how AddExitTask is called
+# (see AddExitTask in _input-output-functions.sh) the ordering of how AddExitTask is called
+# must begin with the to-be-last-run exit task and end with the to-be-first-run exit task:
+if test "$( echo $VAR_DIR/restore/* )" ; then
+    copy_restore_log_exit_task="mkdir $recovery_system_recover_log_dir/restore && cp -pr $VAR_DIR/restore/* $recovery_system_recover_log_dir/restore || true"
```

Comment:
I suggest to use `mkdir -p` this will avoid to black the copy if the directory is already created.